src/services/DappsStakingService.ts

```diff
@@ -0,0 +1,56 @@
+import { injectable, inject } from 'inversify';
+import { IApiFactory } from '../client/ApiFactory';
+import { AprCalculationData } from '../models/AprCalculationData';
+import { networks } from '../networks';
+import { defaultAmountWithDecimals, reduceBalanceToDenom } from '../utils';
+
+export interface IDappsStakingService {
+    calculateApr(network?: string): Promise<number>;
```

Comment:
For static typing, I recommend using:
```ts
type NetworkType = 'astar' | 'shiden';
```

or use an enum instead of a string

Reply:
fixed